pkg/security/probe/probe_ebpf.go

```diff
@@ -1939,7 +1939,7 @@ func (p *EBPFProbe) HandleActions(ctx *eval.Context, rule *rules.Rule) {
 			sig := model.SignalConstants[action.Kill.Signal]
 
 			for _, pid := range pids {
-				if pid == 0 || pid == utils.Getpid() {
+				if pid <= 2 || pid == utils.Getpid() {
```

Comment:
I'm not sure what's the intent behind the `pid <= 2` check instead of checking  for `pid <= 1`. If the goal is to skip `kthreadd` then I'm fine with this.